presto-orc/src/main/java/com/facebook/presto/orc/stream/LongInputStreamV1.java

```diff
@@ -34,6 +34,26 @@
     private boolean repeat;
     private long lastReadInputCheckpoint;
 
+    // ========== ARIA FIELDS BEGIN ==========
```

Comment:
These comments are not useful unless used consistently throughout. Let's remove.